packages/logging/src/errorTransformer.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * The purpose of this file is to provide redux-logger middleware for error logging
+ * To be applied as errorTransformer, or even actionTransformer
+ * @exports redux-logger compatible error logging middleware
+ **/
+
+import TraceKit from 'tracekit';
+import 'whatwg-fetch';
+
+import { sendReport, getDefaultTransport } from './transport';
+
+const defaultOptions = {
+	stackTraceLimit: Error.stackTraceLimit,
+	linesOfContext: 15,
+	rethrowError: false,
+};
+
+/**
+ * @param fallback - original TraceKit.report method
+ * @returns Report, wrapped TraceKit.report method which no longer rethrows an error
+**/
+function safeWrapReport() {
+	function report(ex) {
+		try {
+			TraceKit.report(ex);
+		} catch (e) {
+			// TraceKit throws error bringing down current application, but we don't want that
+			// currently there is no way to fix it with some TraceKit option, thus monkey-patching
```

Comment:
Could you at least handle the exception by adding a console.error log or something?